components/Button/button.css

```diff
@@ -0,0 +1,123 @@
+/* @import url("https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap");
+
+body {
+  font-family: "Roboto", sans-serif;
+} */
+
+:root {
+  --primary-color: #ff6d6d;
+  --disabled-bg-color: #f88e8e;
+  --secondary-color: rgb(213, 213, 213);
+  --secondary-txt-color: rgb(80, 80, 80);
+}
+
+.button_main_container {
+  margin: 1rem 0.5rem;
+}
+
+.btn_main_container {
+  border: 1px solid lightgray;
+  border-radius: 10px;
```

Comment:
Maintain unit consistency

Reply:
Yup. So basically I just use px for border and border-radius.